incskill_site/static/css/resources.css

```diff
@@ -0,0 +1,70 @@
+#btns{
+  position: relative;
+  height:5%;
+}
+
+#sec-btn {
+  margin:auto;
+  width:30%;
+  border-radius:50px;
+  background:#233854;
+  color: white;
+  font-family: Arial, Helvetica, sans-serif;
+  border-radius: 75px;
+  text-decoration: none;
+  cursor: pointer;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding-top: 1em;
+  margin-bottom:1%;
+}
+
+header{
+  border-radius: 0.5%;
+  color: white;
+  background: radial-gradient(39.66% 139.43% at 8.13% 94.24%, rgba(0, 135, 193, 0.2) 0%, rgba(0, 0, 0, 0) 100%) /* warning: gradient uses a rotation that is not supported by CSS and may not behave as expected */, radial-gradient(47.57% 69.38% at 63.57% -5.91%, rgba(60, 180, 188, 0.2) 0%, rgba(51, 154, 160, 0) 90.1%) /* warning: gradient uses a rotation that is not supported by CSS and may not behave as expected */, radial-gradient(32.76% 54% at 83.93% 93.12%, rgba(249, 185, 246, 0.2) 0%, rgba(249, 185, 246, 0) 100%) /* warning: gradient uses a rotation that is not supported by CSS and may not behave as expected */, radial-gradient(200.18% 422.3% at 2.4% 4.39%, #090D2A 0%, rgba(12, 15, 42, 0) 100%) /* warning: gradient uses a rotation that is not supported by CSS and may not behave as expected */, radial-gradient(132.32% 231.51% at 92.86% 14.21%, #080D32 0%, rgba(8, 13, 50, 0) 100%) /* warning: gradient uses a rotation that is not supported by CSS and may not behave as expected */;
+}
+
+#sec{
+  font-family: Arial, Helvetica, sans-serif;
+  margin-left: 3%;
+  margin-right: 3%;
+  margin-bottom: 3%;    
+  font-size: medium;
+  font-weight: 500;
+}
+
+#sec-h1{
```

Comment:
do u even need a specific id or class for this? why not just select the h1's inside of a `sec` by doing `#sec h1 {` (or if you do end up changing it, `.sec h1 {`) then you can remove the extra `#sec-h1` id from your html templates